src/misc/cache/lv_image_header_cache.h

```diff
@@ -35,24 +33,29 @@ extern "C" {
  * Initialize image header cache.
  * @return LV_RESULT_OK: initialization succeeded, LV_RESULT_INVALID: failed.
  */
-lv_result_t lv_image_header_cache_init(void);
+lv_result_t lv_image_header_cache_init(uint32_t count);
 
 /**
  * Resize image header cache.
+ * If set to 0, the cache is disabled.
  * @param new_size  new size of the cache in count of image headers.
  * @param evict_now true: evict the image headers should be removed by the eviction policy, false: wait for the next cache cleanup.
  */
 void lv_image_header_cache_resize(uint32_t new_size, bool evict_now);
 
-#endif /*LV_IMAGE_HEADER_CACHE_DEF_CNT > 0*/
-
 /**
  * Invalidate image header cache. Use NULL to invalidate all image headers.
  * It's also automatically called when an image is invalidated.
  * @param src pointer to an image source.
  */
 void lv_image_header_cache_drop(const void * src);
 
+/**
+ * Return true if the image header cache is disabled.
+ * @return true: disabled, false: enabled.
+ */
+bool lv_image_header_cache_is_disabled(void);
```

Comment:
Please use positive logic, not negative:
```suggestion
bool lv_image_header_cache_is_enabled(void);
```